jboss-eap-standalone-rhel7/metadata.json

```diff
@@ -4,8 +4,8 @@
   "itemDisplayName": "JBoss EAP server running a test application called dukes",
   "description": "This template allows you to create an Red Hat VM running JBoss EAP 7 and and also deploy a web application called dukes, you can login into the admin console using the user and password configured at the time of the deployment.",
   "summary": "Deploy a web application to test Red Hat VM running JBoss EAP7 in standalone mode",
-  "githubUsername": "myriamfentanes",
-  "dateUpdated": "2018-07-31"
+  "githubUsername": "danieloh30",
```

Comment:
do you want to take ownership of this sample?

Reply:
Sure. I think that it would be better to maintain by Red Hat technical marketing team where I'm working.

Reply:
ok, we'll need to be able to run a manual CI test - lmk what's needed for that.

Reply:
Cool. Thx